src/wallet/rpc/wallet.cpp

```diff
@@ -234,8 +234,21 @@ static RPCHelpMan loadwallet()
                     }
                 },
                 RPCExamples{
-                    HelpExampleCli("loadwallet", "\"test.dat\"")
-            + HelpExampleRpc("loadwallet", "\"test.dat\"")
+                    "\nLoad wallet with files under wallets/foo/:\n"
+                    + HelpExampleCli("loadwallet", "\"foo\"")
```

Comment:
```suggestion
                    + HelpExampleCli("loadwallet", "\"walletname\"")
```
_walletname_ is easier to understand than _foo_ imo

Reply:
Done